docs/modules/ROOT/nav.adoc

```diff
@@ -61,6 +61,7 @@ include::user:ROOT:partial$shared-nav.adoc[]
 ** xref:how-to/using-templates.adoc[Using OpenShift Templates]
 ** xref:how-to/grant-access.adoc[Granting Access]
 ** xref:how-to/check-cpu-requests.adoc[Investigate and resolve fair use ratio violations]
+** xref:how-to/use-vpa.adoc[]
```

Comment:
What will be the text of this link?

Reply:
Antora takes the title of the file automatically, no need to repeat it in `nav.adoc` (don't know since when this is the case, but it works like that now)